tools/cldr-apps/js/src/esm/cldrReport.js

```diff
@@ -181,61 +151,42 @@ async function reportTypes() {
   return (await client.apis.voting.listReports()).body.sort();
 }
 
-async function downloadAllReports(user) {
-  const { types, raw, byLocale } = await fetchAllReports(user);
+async function downloadAllReports() {
+  const { types, byLocale } = await fetchAllReports();
 
   const wb = XLSX.utils.book_new();
 
-  var ws_name = `SurveyTool Report on Reports ${user}`;
+  var ws_name = `SurveyTool Report on Reports`;
 
   /* make worksheet */
-  var ws_data = [
-    [
-      "Locale", // 0
-      "Code", // 1
-      "Completion", // 2
-      "Acceptable", // 3
-      "Unacceptable", // 4
-      "Total Votes",
-    ],
-  ];
+  var ws_data = [["Locale", "Code", "Total Votes"]];
```

Comment:
best practice is to use "let" instead of "var", the latter being bug-prone and rarely having any advantage -- in this case it looks like "const" would be even better

also this function is 50 lines long, should be shortened with subroutines; no need for pushComment to be nested; short simple functions with descriptive names are more maintainable

Reply:
@btangmu  done and improved.  This function is a spreadsheet export, I think it would hurt readability to split it up further, as it would make it harder to follow the flow of the data generation.